docs/source/tutorials/multifidelity/mf_pasha.rst

```diff
@@ -0,0 +1,89 @@
+ The experimental comparison shows that PASHA identifies well-performing hyperparameter
+ configurations and architectures while consuming significantly fewer computational
+ resources than ASHA.
+
+What is PASHA?
+--------------------------------------------
+The goal of `PASHA <https://openreview.net/forum?id=syfgJE6nFRW>`_ is to identify
+ well-performing configurations significantly faster than current methods,
+ so that we can then retrain the model with the selected configuration
+ (in practice on the combined training and validation sets). By giving preference
+ to evaluating more configurations rather than evaluating them for longer than
+ needed, PASHA can lead to significant speedups while achieving similar performance
+ as existing methods.
+
+PASHA is a variant of ASHA that starts with a small amount of initial resources
+ and gradually increases them depending on the stability of configuration rankings
+ in the top two rungs (rounds of promotion). Each time the ranking of configurations
+ in the top two rungs becomes inconsistent, PASHA increases the maximum number
+ of resources. This can be understood as "unlocking" a new rung level. An illustration
+ of how PASHA stops early if the ranking of configurations has stabilized
+ is shown in Figure 1.
+
+.. image:: img/pasha_illustration.png
+   :width: 768 px
+
+Given that deep-learning algorithms typically rely on stochastic gradient descent, ranking 
+inconsistencies can occur between similarly performing configurations. Hence, we need some 
+benevolence in estimating the ranking. As a solution, PASHA uses a soft-ranking
+ approach where we group configurations based on their validation performance metric
+ (e.g. accuracy).
+
+In soft ranking, configurations are still sorted by predictive performance but they
+ are considered equivalent if the performance difference is smaller than a
+ value :math `\epsilon` (or equal to it). Instead of producing a sorted list of configurations,
+ this provides a list of lists where for every position of the ranking there is a list
+ of equivalent configurations. The concept is explained graphically in Figure 2. 
+The value of :math `\epsilon` is automatically estimated by measuring noise in rankings.
+
+.. image:: img/soft_ranking.png
+   :width: 768 px
+
+How well does PASHA work?
+--------------------------------------------
+
+We use NASBench201 to compare PASHA with ASHA (promotion type) and the relevant baselines
+ - one-epoch and random - to evaluate how well PASHA works.
+The results suggest that PASHA consistently leads to strong improvements in runtime,
+ while achieving similar accuracies as ASHA. The one-epoch baseline has noticeably
+ worse accuracies than ASHA or PASHA, suggesting that PASHA does a good job of deciding
+ when to continue increasing the resources. Random baseline
+ is a lot worse than the one-epoch baseline, so there is value in performing NAS.
+
+.. image:: img/pasha_results.png
```

Comment:
You can show results in your tutorial if you like, but I'd rather have a link to the paper.

Reply:
I've removed the table and put there a short summary about the speedups, with a link to the paper for more details